### DISCUSSION TITLE
cheat codes

### DISCUSSION THREAD

**Discussion by Quantaviousdingleberry420** [aeolus-1/triangleGame.github.io#30] [Ideas]
reset - will reset the game and put you at the start (multiplayer wont move)
ghost - will disable collision for one layer, useful for finding easter eggs

add any more in comments or gay

**Comment by aeolus-1** [comment]
joinpl - adds a new player

**Comment by nekoify** [comment]
sussy balls - teleports you to the end

**Comment by Bruce-Browninggg** [comment]
None of these work.

**Reply by nekoify** [reply]
they disabled
